wdl/qc_imputation.wdl

```diff
@@ -271,7 +272,10 @@ task vcf_to_bed {
         <(zcat -f ${vcf} | grep -m1 -E "^#CHROM" | tr '\t' '\n' | tail -n+10 | grep -E "${include_regex}" | sort) \
         ${exclude_samples} > ${base}.exclude_samples_upfront.txt
 
+        touch dummy_exclude_probes.txt
+
         bcftools view -S include_samples.txt ${vcf} -Ov | \
+        grep -Fwvf ${default='dummy_exclude_probes.txt' exclude_probes} | \
```

Comment:
this is fine here with -w switch but always a bit error prone to grep on whole lines. Would usually be good to check only the column in question... amazing btw. that bcftools view does not have variant exclude option.

Can you create r12 chip qc json and add this new option there to remove chosen variants....

Reply:
this would be one option to safeguard against general grepping

 awk 'BEGIN{FS=OFS="\t"} NR==FNR{ exclude[$1]=1 } NR>FNR&&($1 ~ /^#/ || !($3 in exclude)){ print }' excl t.vcf

Reply:
Oh but there is an exclude option in bcftools view.... Much simpler with that.

Btw with the awk command NR==FNR check is no good with empty files as in the `touch dummy_exclude_probes.txt`, would need something like `echo -e "\n" > dummy_exclude_probes.txt`. Good to remember going forward.

Reply:
ah good... whats the exclude option.?..  oh good point about 0 line file!!!!